marimo/_ast/app.py

```diff
@@ -54,7 +54,7 @@ class _AppConfig:
     a single marimo program.
     """
 
-    width: WidthType = "normal"
+    width: WidthType = "medium"
```

Comment:
i change this default since the user config's default is medium? this wouldnt change much but i figured worth being consistent?

Reply:
The only thing is that it would change everyone's existing apps, for those who relied on default being normal (since we didn't serialize the default config). That's why I left it as is